meas/MeasUDF/test/azel.cpp

```diff
@@ -0,0 +1,39 @@
+//# azel.cc: utility to print azimut and elevation of a given source
```

Comment:
The standard casacore header has been omitted

Reply:
Valid point. Will ignore for now.